packages/chart/src/isFigureChartModel.ts

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
```

Comment:
Get rid of these compiler warnings. There's an easy way to get `FigureChartModel` to conform to `ChartModel` without having to convert everything immediately.